### PR TITLE
SERVER-75247 Added mongodb 7.0 release

### PR DESCRIPTION
**Pull request by IamXander**

### CODE REVIEW

File: Formula/mongodb-community@6.0.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+class MongodbCommunity < Formula
```
Comment (IamXander) [inline Formula/mongodb-community@6.0.rb:1]: This file is unchanged just renamed

File: Formula/mongodb-community.rb [hunk 2]
```diff
@@ -5,11 +5,11 @@ class MongodbCommunity < Formula
   # frozen_string_literal: true
 
   if Hardware::CPU.intel?
```
Comment (IamXander) [inline Formula/mongodb-community.rb:7]: This file is a copy of `Formula/mongodb-community@6.0.rb` with the diff being the only thing that changed

File: Formula/mongodb-enterprise@6.0.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
+class MongodbEnterprise < Formula
```
Comment (IamXander) [inline Formula/mongodb-enterprise@6.0.rb:1]: ditto

File: Formula/mongodb-mongocryptd.rb [hunk 4]
```diff
@@ -3,11 +3,11 @@ class MongodbMongocryptd < Formula
   homepage "https://www.mongodb.com/"
 
   if Hardware::CPU.intel?
```
Comment (IamXander) [inline Formula/mongodb-mongocryptd.rb:5]: ditto

File: Formula/mongodb-mongocryptd@6.0.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+class MongodbMongocryptd < Formula
```
Comment (IamXander) [inline Formula/mongodb-mongocryptd@6.0.rb:1]: ditto

**Review by Trevor159 — APPROVED** [review]

**Comment by Hackerbee** [comment]
Couldn't find the issues tab so commenting here.

`brew install mongodb-community@6.0` is broken.  I resolved it by modifying the formula as mentioned in the error, by replacing `MongodbCommunity` with `MongodbCommunityAT60`.

```
❯ brew install mongodb-community@6.0              
Error: No available formula with the name "mongodb-community@6.0". Did you mean mongodb-community@5.0, mongodb-community or mongodb-community@4.4?
In formula file: /opt/homebrew/Library/Taps/mongodb/homebrew-brew/Formula/mongodb-community@6.0.rb
Expected to find class MongodbCommunityAT60, but only found: MongodbCommunity.
```

File: Formula/mongodb-community@6.0.rb (hunk 1 above)
Comment (Hackerbee) [reply Formula/mongodb-community@6.0.rb:1]: This needs to change to `MongodbCommunityAT60`

File: Formula/mongodb-enterprise@6.0.rb (hunk 3 above)
Comment (Hackerbee) [reply Formula/mongodb-enterprise@6.0.rb:1]: Maybe this as well. I don't use mongodb enterprise.

**Comment by IamXander** [comment]
@Hackerbee thank you for the note: this is fixed in https://github.com/mongodb/homebrew-brew/pull/194

**Pull request merged**